operations/helm/charts/mimir-distributed/templates/lib/map-to-csv.tpl

```diff
@@ -0,0 +1,12 @@
+{{/*
+Convert labels to string like: key1="value1", key2="value2", ...
```

Comment:
Is it `key1="value1"` or `key1=value1`? Also I think there isn't any space between the keys. Can you add an example input and output in the comment?